lib/modules/dosomething/dosomething_reportback/dosomething_reportback.forms.inc

```diff
@@ -371,24 +371,59 @@ function dosomething_reportback_form_submit($form, &$form_state) {
     $values['uid'] = $user->uid;
   }
 
-  // Save uploaded file.
-  if ($file = dosomething_reportback_form_save_file($form_state)) {
-    // Store new file's fid into values.
-    $values['fid'] = $file->fid;
-  }
+  if (module_exists('dosomething_rogue')) {
```

Comment:
true! Updated to check for the flag we added in the custom settings form. This is per node for now. If we move forward with wanting this on all campaigns we should probably switch this to a more global feature flag.